### ISSUE TITLE
Code snippets use " instead of '

### ISSUE THREAD

**Issue by mortomanos** [awatson1978/meteor-api#73]
The code snippets use a double quote instead of a single quote which is preferred by eslint.

**Comment by awatson1978** [comment]
Yeah, this has come to my attention; and I spent some time trying to reconcile the snippets, linters, and code highlighters.  Unfortunately, I don't think it's possible to get it all consistent.  The code is accessed in different interpretation contexts (the entire reason that ``eval()`` exists in the first place), and setting a universal linting rule is like arguing against a diagonal proof.  It's like a mobius strip.  If I move the code snippets to use single quotes, it will cause doublequotes complaints elsewhere.  

I think my solution moving forward will be to remove the rule that detects singlequote vs doublequote from the default starrynight ``.eslint`` file.
